### PR TITLE
SDSS-364: Sync people displays with stanford_profile upstream

### PR DESCRIPTION
**Pull request by joegl**
# READY FOR REVIEW

# Summary
- We customized the People list view and display in `sdss_profile` so it diverged from `stanford_profile`. `stanford_profile` made updates to this view, including using `<h2>` and `<h3>` for the people titles and tests for the headers. These tests fail in `sdss_profile` because of the divergence in the view and not inheriting the header updates.
- This PR syncs the view with `stanford_profile`, getting the updates, but also restoring the customization made in the `sdss_profile`

# Review By (Date)
ASAP

# Urgency
HIGH

# Review Tasks

## Setup tasks and/or behavior to test
1. Check out the `SDSS-362--update-sdss-profile` branch for `ace-sdssgryphon` and run `composer install`
2. Run `composer require su-sws/sdss_profile:SDSS-364--people-list-heading-dev` to get this branch for `sdss_profile`
3. Rebuild Cache and import config `drush cr ; drush ci`
4. Navigate to `/people`
5. Verify the people titles are now in `<h2>` and the styles match what's on the sustainability site.
6. Click on a Person Type in the left hand sidebar until you get a page with a person on it.
7. Verify the people titles are now in `<h3>` and the styles are similar to the sustainability site (don't have to match).

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? **Yes**

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
https://stanfordits.atlassian.net/browse/SDSS-362
https://stanfordits.atlassian.net/browse/SDSS-364

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by joegl** [comment]
@pookmish I just added you to take a quick peek, no need for full review or anything.

**Review by pookmish — APPROVED** [review]

**Comment by jenbreese** [comment]
The font size for the `h3 ` on the filtered page is a smaller size.

File: themes/sdss_subtheme/src/scss/components/people/_people.scss [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
 .people-terms {
   .views-field-title {
     a {
       @include sdss-h4;
       @include sdss-link--primary;
+
+      h2,
+      h3 {
+        display: inline;  // Necessary to get the outline to work correctly on focus.
+        font-weight: 400;
```
Comment (jenbreese) [inline themes/sdss_subtheme/src/scss/components/people/_people.scss]: You could use this variable here for the 400: $su-font-regular

**Review by jenbreese — CHANGES_REQUESTED** [review]
I approve with the variable change.

**Pull request merged**